donut_shop.html

```diff
@@ -5,33 +5,25 @@
 </head>
 <body bgcolor="#FACC2Eg">
   <h1>DonutShop</h1>
-  <table style="width:100%">
+  <table id= "tbl" style="width:100%">
     <h1><center><font color="#FF0080"> Top Pot Donuts Works for You!</center></h1>
```

Comment:
You're learning CSS this week, but I want to point out that from here forward you should not be styling your page from HTML elements. :-)  Use those new CSS super powers!